### PR TITLE
Syndicate Uplink - Typos & Grammar mk2

### PR DESCRIPTION
**Pull request by ShizCalev**
Now Queen's English friendly.

### CODE REVIEW

**Comment by banthebantz** [comment]
**The only damned 'English' I recognise is that of the Queen!**

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -890,8 +890,8 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	cost = 1
 
 /datum/uplink_item/device_tools/surgerybag
-	name = "Syndicate Surgery Dufflebag"
-	desc = "The Syndicate surgery dufflebag is a toolkit containing all surgery tools, surgical drapes, \
+	name = "Syndicate Surgery Duffle bag"
```
Comment (Iamgoofball) [inline code/modules/uplink/uplink_item.dm]: looks weird tbh

**Comment by ShizCalev** [comment]
Dufflebag is not actually a word, sadly. Duffle bag is the correct term.

**Comment by Iamgoofball** [comment]
remove brits tbh

**Comment by Iamgoofball** [comment]
make it gym bag then

so i can show you who's boss of this language

**Comment by Tacolizard** [comment]
you changed the s's in specialized to z's. That's how we spell it in America, so I don't think that's 'queen's English' friendly.

**Comment by ShizCalev** [comment]
@Tacolizard The Oxford English dictionary states that both are acceptable British spellings. Specialised is an older form of the spelling that is growing out of use, and specialized is more commonplace now.

https://en.oxforddictionaries.com/definition/specialized

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -455,22 +455,22 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	include_modes = list(/datum/game_mode/nuclear)
 
 /datum/uplink_item/ammo/shotgun/bag
-	name = "12g Ammo Duffelbag"
-	desc = "A duffelbag filled with enough 12g ammo to supply an entire team, at a discounted price."
+	name = "12g Ammo Duffel bag"
```
Comment (lzimann) [inline code/modules/uplink/uplink_item.dm]: We use dufflebag in the game though, so this will be inconsistent

Comment (ShizCalev) [inline code/modules/uplink/uplink_item.dm]: Inconsistent actually. There are references to both duffelbag and dufflebag.

Comment (ShizCalev) [inline code/modules/uplink/uplink_item.dm]: Will switch it over. Thought that spelling was consistent elsewhere and that duffle was the outlier.

**Comment by ShizCalev** [comment]
honk, sec.

**Comment by ShizCalev** [comment]
Will save that one for another day then.

**Comment by lzimann** [comment]
I don't really care how is it spelled, as long as it is the same everywhere!

**Comment by ShizCalev** [comment]
It's consistent now. The inconsistencies that were already in the uplink threw me off on which was used elsewhere.

**Pull request merged**